canonical-websites/www.ubuntu.com
# Created the engage snap deltas page

## Done

- Created a new /engage/snap-deltas page
- Created a copy doc
- Tweaked the header include to have a width and a max-width and max-height

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/snap-deltas](http://0.0.0.0:8001/engage/snap-deltas)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1BzilE80HtJ89mPXnhkaK0AYyYuPr8fnJPwnVUaMhJ9Y/edit#) and test the form

## Issue / Card

Fixes #4834
